index.js

```diff
@@ -1,9 +1,9 @@
-const alex = require('alex')
 const mediaType = 'application/vnd.github.antiope-preview+json'
 const headers = {headers: {accept: mediaType}}
 
 module.exports = (robot) => {
   robot.on('check_suite', async context => {
+    const analyzeTree = require('./lib/analysis')(context)

```

Comment:
Move this require to top-level, so that it resolves it when the server starts, not when the handler is run.

```js
const analyzer = require('./lib/analysis')

robot.on('check_suite', async context = {
  const analyzeTree = analyzer(context)
}
```